samples/oauth-sample/ui/src/App.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Button from "@mui/material/Button";
+import { createDockerDesktopClient } from "@docker/extension-api-client";
+import { Stack, TextField, Typography } from "@mui/material";
+
+const client = createDockerDesktopClient();
+
+const client_id = "xxxx";
+const client_secret = "xxxx";
```

Comment:
** suggestion:** Could you add instructions in the readme so that the sample works out of the box?

Reply:
Added in the readme